### ISSUE TITLE
Use monospace for Advanced, Additional Attributes

### ISSUE THREAD

**Issue by emrecio** [keepassxreboot/keepassxc#10747]
## Summary
[TIP]:  # Edit Entry, Select Advanced in Left Panel, Add an "Additional Attribute"
[NOTE]: # Just like the Notes section, it would be nice to have it in monospace so that you can enter multi-line data (e.g.: numeric values, etc.) and have them all line up correctly.

## Examples
[NOTE]: # Current vs Proposed, the following is the same amount of text with identical spacing

foo 123 23
bar  34 11
baz 099 88

```
foo 123 23
bar  34 11
baz 099 88
```

## Context
[NOTE]: # I store ancillary data related to the main entry where monospace may makes reading the column aligned data easier. Further, this may be used to store backup codes and make them more legible, etc.

**Comment by scollovati** [comment]
Agree. This is useful also for Passkeys data that are stored via Additional Attributes.

The good thing is that the HTML export already prints them with a monospaced font.
